### PR TITLE
fix whitespace issues throughout codebase

### PR DESCRIPTION
**Pull request by sbromberger**

### CODE REVIEW

File: src/LightGraphs.jl [hunk 1]
```diff
@@ -86,9 +86,6 @@ CombinatorialAdjacency, non_backtracking_matrix, incidence_matrix,
 nonbacktrack_embedding, Nonbacktracking,
 contract,
 
-# astar
-a_star,
```
Comment (sbromberger) [inline src/LightGraphs.jl]: It's defined twice.

Comment (sbromberger) [inline src/LightGraphs.jl]: (This was one of I think three non-whitespace changes I made [prior to our IM conversation last night]).

Comment (jpfairbanks) [inline src/LightGraphs.jl]: ok cool

File: src/generators/euclideangraphs.jl [hunk 2]
```diff
@@ -35,23 +35,23 @@ function euclidean_graph(points::Matrix;
...
-    cutoff < 0. && (cutoff=typemax(Float64))
+    cutoff < 0. && (cutoff = typemax(Float64))
     if bc == :periodic
-        maximum(points) > L &&  error("Some points are outside the box of size $L.")
+        maximum(points) > L && error("Some points are outside the box of size $L.")
     end
-    for i=1:N
-        for j=i+1:N
+    for i = 1:N
+        for j = (i + 1):N
```
Comment (jpfairbanks) [inline src/generators/euclideangraphs.jl]: I think this is uglier than i+1:N but I guess it is more explicit so we should use it.

Comment (sbromberger) [inline src/generators/euclideangraphs.jl]: It is, but it's explicit, as you said. There is a fundamental difference between
```
for j = x : y + z
```
and
```
for j = (x : y) + z
```
and I'm trying to make sure we don't get bitten by ambiguities.

**Comment by sbromberger** [comment]
Thanks for the review. I'll merge now.

**Pull request merged**